src/lib/eZ/RichText/Resources/stylesheets/xhtml5/edit/docbook.xsl

```diff
@@ -9,6 +9,9 @@
     version="1.0">
   <xsl:output indent="yes" encoding="UTF-8"/>
 
+  <xsl:key name="ids" match="*[@id]" use="@id"/>
+  <xsl:key name="ids" match="ezxhtml5:a[@name]" use="@name"/>
```

Comment:
Why both conditions are matched by the same `name="ids"`? Just asking, I am not familiar with such notation but maybe it does the job just fine.

Reply:
We are scanning for IDs and putting them in the same map (which is named {{ids}}). 

So we have html5 edit code like this:
`<p><a name="bottom"/>another anchor with the same name attribute.</p>`
where the value of the `name` attribute will also end up as an id, see [[xhtml5/edit/lossy/008-anchor.docbook.xml test](https://github.com/ezsystems/ezplatform-richtext/pull/243/files#diff-5dcffba4a6512771e0cca2bf0b7d65bfea0df58a748c3be6a1e92b9c0642ebc1)

Reply:
Thank you, that makes sense.